test/spec/modules/vertozBidAdapter_spec.js

```diff
@@ -0,0 +1,64 @@
+import { expect } from 'chai';
+import { spec } from 'modules/vertozBidAdapter';
+import { newBidder } from 'src/adapters/bidderFactory';
+
+const BASE_URI = '//hb.vrtzads.com/vzhbidder/bid?';
+
+describe('VertozAdapter', () => {
+  const adapter = newBidder(spec);
+
+  describe('inherited functions', () => {
+    it('exists and is a function', () => {
+      expect(adapter.callBids).to.exist.and.to.be.a('function');
+    });
+  });
+
+  describe('isBidRequestValid', () => {
+    let bid = {
+      'bidder': 'vertoz',
+      'params': {
+        'placementId': 'VZ-HB-B784382V6C6G3C'
+      },
+      'adUnitCode': 'adunit-code',
+      'sizes': [[300, 250], [300, 600]],
+      'bidId': '30b31c1838de1e',
+      'bidderRequestId': '22edbae2733bf6',
+      'auctionId': '1d1a030790a475',
+    };
+
+    it('should return true when required params found', () => {
+      expect(spec.isBidRequestValid(bid)).to.equal(true);
+    });
+
+    it('should return false when required params are not passed', () => {
+      let bid = Object.assign({}, bid);
+      delete bid.params;
+      bid.params = {
+        'placementId': 0
+      };
+      expect(spec.isBidRequestValid(bid)).to.equal(false);
+    });
+  });
+
+  describe('buildRequests', () => {
+    let bidRequests = [
+      {
+        'bidder': 'vertoz',
+        'params': {
+          'placementId': '10433394'
+        },
+        'adUnitCode': 'adunit-code',
+        'sizes': [[300, 250], [300, 600]],
+        'bidId': '30b31c1838de1e',
+        'bidderRequestId': '22edbae2733bf6',
+        'auctionId': '1d1a030790a475',
+      }
+    ];
+
+    it('sends bid request to ENDPOINT via POST', () => {
+      const request = spec.buildRequests(bidRequests)[0];
+      expect(request.url).to.equal(BASE_URI);
+      expect(request.method).to.equal('POST');
+    });
+  });
+});
```

Comment:
These tests are only providing 68.97% coverage of the adapter, we require at least 80% before merge. Removing the unneeded code mentioned in the other comments should bump the percentage up, and then testing the `interpretResponse` should help getting there